### DISCUSSION TITLE
Trying to use imgui.Separator() but getting error.

### DISCUSSION THREAD

**Discussion by vbmark** [AllenDang/giu#650] [Q&A] [answered]
Hello,

I am not sure if what I am doing here is possible or not but I gave it a try and am getting this error:

![image](https://user-images.githubusercontent.com/89524/222929885-e4d556b7-c39b-4f34-98a1-67ba2baa8672.png)

I added this import `"github.com/AllenDang/imgui-go"`

What am I doing wrong?

Thank you.

EDIT: Never mind.  I just changed `imgui.Separator(),` to `g.Separator(),` and it works great.

This library is truly amazing!

**Comment by gucio321** [comment, marked as answer]
> EDIT: Never mind. I just changed imgui.Separator(), to g.Separator(), and it works great.

just one word of explaination: imgui.Separator (as an imgui function) is of type `func()`, and giu.Separator creates giu.SeparatorWidget, which implements `giu.Widget` interface
